### PR TITLE
interest-is-interesting: explain interest

### PR DESCRIPTION
**Pull request by dzegel**
Adding basic context on compounding interest.

### CODE REVIEW

File: exercises/concept/interest-is-interesting/.docs/introduction.md [hunk 1]
```diff
@@ -41,3 +41,10 @@ do
     x = x - 2;
 } while (x > 10)
 ```
+
+## Simple Interest & Compounding Interest
+
+When applying simple interest to a principle balance, the balance is multipled by the interest rate and the product of the two is the interest amount.
+
+Compound interest on the other hand is done by applying interest on a reocurring bases.
+On each application the interest amount is computed and added to the principle balance so that subesquent interest calculations are subject to a greater principle balance.
```
Comment (ErikSchierboom) [inline exercises/concept/interest-is-interesting/.docs/introduction.md:50]: The introduction is there just to explain the language concepts the exercise teaches. Could you move this to the corresponding task? You could also consider making it a special block, see [the docs](https://exercism.org/docs/building/markdown/markdown#h-special-blocks-sometimes-called-admonitions).

**Review by ErikSchierboom — COMMENTED** [review]
Thanks! I've left a comment.

**Comment by dzegel** [comment]
@ErikSchierboom any other concerns?

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/concept/interest-is-interesting/.docs/instructions.md [hunk 2]
```diff
@@ -44,11 +44,18 @@ Note that the value returned is a `decimal`.
...
 SavingsAccount.YearsBeforeDesiredBalance(balance: 200.75m, targetBalance: 214.88m)
 // 14
 ```
 
 Note that the value returned is an `int`.
+
+~~~~exercism/note
+When applying simple interest to a principle balance, the balance is multipled by the interest rate and the product of the two is the interest amount.
+
+Compound interest on the other hand is done by applying interest on a reocurring bases.
```
Comment (ErikSchierboom) [inline exercises/concept/interest-is-interesting/.docs/instructions.md:59]:
```suggestion
Compound interest on the other hand is done by applying interest on a recurring basis.
```

**Review by ErikSchierboom — COMMENTED** [review]
One tiny nit.

File: exercises/concept/interest-is-interesting/.docs/introduction.md [hunk 3]
```diff
@@ -41,3 +41,10 @@ do
     x = x - 2;
 } while (x > 10)
 ```
+
+## Simple Interest & Compounding Interest
+
+When applying simple interest to a principle balance, the balance is multipled by the interest rate and the product of the two is the interest amount.
```
Comment (okaram) [inline exercises/concept/interest-is-interesting/.docs/introduction.md]: Principal, not principle (a common mistake)

Comment (dzegel) [reply exercises/concept/interest-is-interesting/.docs/introduction.md]: oh me and spelling, thanks for pointing this out

**Comment by dzegel** [comment]
@ErikSchierboom I think this is ready to merge?

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Yep, merged! 🎉

**Pull request merged**